run_sweep.py

```diff
@@ -0,0 +1,30 @@
+import wandb
+import os
+import train
+import argparse
+
+from train import file_path
+from configs import api_key, example_sweep_config
+
+TUNING_METHOD = 'random'
+N_RUNS_IN_SWEEP = 3
+
+def main():
+
+    def call_training_procedure(config_dict=None):
```

Comment:
Why did you define this function inside of `main`?

Reply:
wandb.agent requires a training function which takes only one argument (the hyperparameter config dictionary). Therefore, train.train_procedure cannot be used directly, as it expects multiple arguments. In order to pass args.train_path, args.val_path and args.test_path to train.train_procedure and at the same time have a function that takes only one argument, I defined this inner function. An alternative would be to put this function outside of main and to put the argparse part inside this function.